crowdsec-docs/docs/observability/usage_metrics.md

```diff
@@ -0,0 +1,61 @@
+---
+id: usage_metrics
+title: Usage Metrics
+sidebar_position: 6
+---
+
+:::info
+Usage metrics require to run CrowdSec v1.6.3.
```

Comment:
`Usage metrics require to run CrowdSec v1.6.3.` -> `Usage metrics require at least CrowdSec v1.6.3`